### PR TITLE
Update link to playlist

### PR DESCRIPTION
**Pull request by alexfoias**
Fixes https://github.com/spinalcordmri/spinalcordmri.github.io/issues/66

### CODE REVIEW

File: _posts/2020-01-20-workshop.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ clinicians from around the world and discuss together and establish new collabor
 
 ### Live streaming
 
-For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). Recordings of the workshop from both days can be found [here](https://www.youtube.com/watch?v=iHbKkL3Kw1I&list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql&index=2). Recording of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).
+For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). Recordings of the workshop from both days can be found [here](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql). Recording of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).
```
Comment (jcohenadad) [inline _posts/2020-01-20-workshop.md:21]: @alexfoias as mentioned in the todo list pls create separate section for recordings

Comment (alexfoias) [reply _posts/2020-01-20-workshop.md:21]: The current link point to the playlist workshops (where all the workshops are listed). You would like a separate playlist called Recordings and migrate the spinal cord workshop 7 videos to it ?

Comment (alexfoias) [reply _posts/2020-01-20-workshop.md:21]: @jcohenadad The [current link](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql) points to the playlist workshops (where all the workshops are listed). You would like a separate playlist called Recordings and migrate the spinal cord workshop 7 videos to it ?

Comment (alexfoias) [reply _posts/2020-01-20-workshop.md:21]: or do you mean section in the actual post that points to the youtube playlist?

Comment (jcohenadad) [inline _posts/2020-01-20-workshop.md:21]:
```suggestion
For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). 

### Recordings
Recordings of the full workshop is available [here](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql). Recordings of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).
```

**Comment by alexfoias** [comment]
sorry for the misunderstanding.

**Pull request merged**